.github/workflows/release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  push-to-release-branch:
+    name: Push source to release branch
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Push to release branch
+      run: |
+        export tag=$(echo ${GITHUB_REF#refs/tags/})
+        export tag_match_pattern=$(echo "$tag" | grep -E "^v[0-9]+\.[0-9]+\.[0-9]+$")
+        export tag_suffix=$(echo ${tag:0:$((${#tag}-1))}x)
```

Comment:
vX.Y.Z
Z (as well as x,y,z) could be multiple digits - if I understand the syntax correctly you only remove the last char

Reply:
Nice catch, fixed